### ISSUE TITLE
Alphabet filter has shuffled list items instead of ASC order.

### ISSUE THREAD

**Issue by udayraj21** [ericferreira1992/alphabet-filter#5]
I have implemented alphabet filter in an angular 7 project. It displays unordered list items. 
Firstly, I checked my API resposene data. 
I found that API returns data in ASC order but not in alphabet filter.
![Screenshot from 2020-03-03 00-24-27](https://user-images.githubusercontent.com/24545222/75707724-72b5ba80-5ce5-11ea-851a-53aba624c036.png)

**Comment by ericferreira1992** [comment]
@udayraj21 Hi!
You can post your case in StackBlitz or another code playground? Maybe you can put fixed data response in the code to we can simulate it case.

Thanks!

**Comment by udayraj21** [comment]
@ericferreira1992 
I have prepared a StackBiz playground. There is no issue with that. There is exactly ASC order.!!!
I am surprised with my project code alphabet filter which behaves weird.!!
I cant found what is the problem with my project code...

You can refer StackBiz code.
https://stackblitz.com/edit/angular-ppyjtb

**Comment by udayraj21** [comment]
Even I have checked with static data in my project. But no luck. It shows shuffled items always.
Exept this page of the project, I have also added alphabet filter in another page. Second page with filter works fine. I cant figure out the exact issue with the first page.

**Comment by ericferreira1992** [comment]
@udayraj21
This is really strange. I don't know what can cause the problem.  Maybe does it have anything to do with using on more than one page, I dont know. But, try do more tests until can found mistake. I has never seen any such behavior.

**Comment by udayraj21** [comment]
@ericferreira1992 
I found the cause of shuffling the list items in alphabet filter finally. Actually API response data was in ASC order but somehow it array items shuffled at the componen render time.
To resolve this issue, I did somthing like below.
In component html file,
`<alphabet-filter propAlphaOrder="name" (input)="filterClients($event)" placeholder="Search clients..." height="600px"
    [data]="clientsListSort()" listClass="search-list"></alphabet-filter>`

And in component ts file,
`public clientsListSort() {
		return this.clientsFilter.sort((a, b) => {
			return a.name.localeCompare(b.name);
		});
	}`
